src/dos/dos_mscdex.cpp

```diff
@@ -253,18 +253,11 @@ int CMscdex::AddDrive(Bit16u _drive, char* physicalPath, Bit8u& subUnit)
 	int result = 0;
 	// Get Mounttype and init needed cdrom interface
 	switch (CDROM_GetMountType(physicalPath,forceCD)) {
-	case 0x00: {	
+	case 0x00:
 		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: Mounting physical cdrom: %s"	,physicalPath);
-#if defined (LINUX)
-		// Always use IOCTL in Linux
-		cdrom[numDrives] = new CDROM_Interface_Ioctl();
-		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: IOCTL Interface.");
-#else
-		// Default case windows and other oses
-		cdrom[numDrives] = new CDROM_Interface_SDL();
-		LOG(LOG_MISC,LOG_NORMAL)("MSCDEX: SDL Interface.");
-#endif
-		} break;
+		// TODO: support for mounting physical CD-ROMs removed, provide
```

Comment:
I like this approach.
We can point user to a README or a Wiki page up-to-date with tools/scripts showing how to backup their game CD-ROM to ISO/FLACs & ISO/Opus tracks.